submissions/2024-11-07-Sage-2.1.0/yds.yaml

```
@@ -0,0 +1,3 @@
+forcefield: openff-2.1.0.offxml
```

```suggestion
forcefield: openff_unconstrained-2.1.0.offxml
```
See recent discussion on Slack - assuming you want to use the unconstrained version, that is

---

Thanks, I think you're right. I'm planning just to let this one run to make sure I didn't break anything with the last PR (I should probably set up some actual test CI soon) and then I'll open another PR with the unconstrained version.